test/MLOps.NET.SQLite.Tests/MLOpsBuilderExtensionTests.cs

```diff
@@ -54,5 +56,24 @@ public void UseSqlLite_ConfiguresLifeCycleCatalog()
             //Assert
             unitUnderTest.LifeCycle.Should().NotBeNull();
         }
+
+        [TestMethod]
+        public void UseSqlLite_ConfiguresAlternativeSQLiteDbPath()
+        {
+            var tempPath = Path.GetTempPath();
+            var tempDbName = Guid.NewGuid().ToString();
+            var tempDbPath = Path.Combine(tempPath, $"{tempDbName}.db");
+
+            //Act
+            IMLOpsContext unitUnderTest = new MLOpsBuilder()
+                .UseSQLite(tempDbPath)
+                .UseModelRepository(new Mock<IModelRepository>().Object)
+                .Build();
+
+            unitUnderTest.Should().BeOfType<MLOpsContext>("Because the default IMLLifeCycleManager is MLLifeCycleManager");
```

Comment:
@WalternativE is there a way we can more accurately test that the path ins correctly set? This test currently only tests that nothing blows up :)

Reply:
To be honest, I think for this change we can actually skip a unit test. Creating a unit test here that adds value is quite a bit of effort which I'm not sure is worth it. But I do like the change, thank you for suggesting this! I'm ready to approve this PR if you could remove the current unit test (hopefully the CI build will pass the next time, if not I can look at what's up) :)

Reply:
I should have guessed, that the test would act up in a CI environment. Hoped `tmp` files were safe but they are not apparently. For cases like this I'd really just want to mock the file system in software somehow but writing this would be absolutely overblown. I'll commit the changes in a sec :) Thanks for looking at them 🙇‍♂️